net/gclient/gclient_tracing_tracer.go

```diff
@@ -153,14 +152,15 @@ func (ct *clientTracer) wroteRequest(info httptrace.WroteRequestInfo) {
 		ct.span.SetStatus(codes.Error, fmt.Sprintf(`%+v`, info.Err))
 	}
 
+	reqBodyContent, err := gtrace.SafeContent(ct.requestBody, ct.request.Header)
+	if err != nil {
+		ct.span.SetStatus(codes.Error, fmt.Sprintf(`safe req content:%+v`, err))
```

Comment:
It is not necessary outputting error stack content to span status.
`ct.span.SetStatus(codes.Error, fmt.Sprintf(`converting safe content failed: %s`, err.String()))`